sdk/storage/azblob/sas/query_params.go

```diff
@@ -23,7 +23,7 @@ const (
 
 var (
 	// Version is the default version encoded in the SAS token.
-	Version = "2020-02-10"
+	Version = "2020-12-06"
```

Comment:
The version should be the STG 85/86 version in my opinion.
```suggestion
	Version = "2021-12-02"
```

Reply:
That's right! Thanks for the call out!